core/interfaces.go

```diff
@@ -135,4 +135,5 @@ type StorageAuthority interface {
 // Publisher defines the public interface for the Boulder Publisher
 type Publisher interface {
 	SubmitToCT(ctx context.Context, der []byte) error
+	SubmitToSingleCT(ctx context.Context, logID string, der []byte) error
```

Comment:
I'm going to leave the publisher's `ctLogs` configuration in-place for this PR so that I don't have to change the API on `SubmitToCT` and break protobuf compatibility. I'll create a follow-up issue to convert `SubmitToCT` to accept a `[]string` of URLs and publicKeys